server/helpers/helpers_test.go

```diff
@@ -0,0 +1,46 @@
+package helpers
+
+import (
+	"os"
+	"testing"
+)
+
+func TestMergeStringMaps(t *testing.T) {
```

Comment:
@ganeshvenkatasai I thought you also added tests for empty maps and merging maps (not only overriding).

Reply:
+1